### PR TITLE
Remove more data streams

### PR DESCRIPTION
**Pull request by KochetovNicolai**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Review by alexey-milovidov — APPROVED** [review]

File: src/Storages/Kafka/KafkaBlockInputStream.cpp [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #include <Storages/Kafka/KafkaBlockInputStream.h>
```
Comment (CurtizJ) [inline src/Storages/Kafka/KafkaBlockInputStream.cpp:1]: Need to rename file?

Comment (CurtizJ) [reply src/Storages/Kafka/KafkaBlockInputStream.cpp:1]: The same for `RabbitMQBlockInputStream.h`.

Comment (KochetovNicolai) [reply src/Storages/Kafka/KafkaBlockInputStream.cpp:1]: Yes

File: src/Storages/Kafka/StorageKafka.cpp [hunk 2]
```diff
@@ -611,14 +609,16 @@ bool StorageKafka::streamToViews()
...
+    sources.reserve(stream_count);
+    pipes.reserve(stream_count);
     for (size_t i = 0; i < stream_count; ++i)
     {
-        auto stream = std::make_shared<KafkaBlockInputStream>(*this, metadata_snapshot, kafka_context, block_io.pipeline.getHeader().getNames(), log, block_size, false);
-        streams.emplace_back(stream);
+        auto source = std::make_shared<KafkaSource>(*this, metadata_snapshot, kafka_context, block_io.pipeline.getHeader().getNames(), log, block_size, false);
+        sources.emplace_back(source);
 
         // Limit read batch to maximum block size to allow DDL
```
Comment (ucasfl) [inline src/Storages/Kafka/StorageKafka.cpp:623]: Forget pipes.emplace_back(source)?

Comment (KochetovNicolai) [reply src/Storages/Kafka/StorageKafka.cpp:624]: Yes, thank you!

**Comment by KochetovNicolai** [comment]
@Mergifyio up

**Comment by KochetovNicolai** [comment]
@mergifyio update

**Review by filimonov — APPROVED** [review]

**Pull request merged**